### PR TITLE
Add inReverse variable to UICircularProgressRing class

### PR DESCRIPTION
**Pull request by jonahollman**
Add inReverse variable to UICircularProgressRing class to all the user to determine which direction the value label is counting.

### CODE REVIEW

File: src/UICircularProgressRing/UICircularProgressRing.swift [hunk 1]
```diff
@@ -135,6 +135,20 @@ final public class UICircularProgressRing: UICircularRing {
...
+     A variable to determine if the value label will count down to zero.
+
+     ## Important ##
+     Default = false
+
+	## Author
+	Jonah Ollman
+	*/
+
+	@IBInspectable public var isReverse: Bool = false {
```
Comment (luispadron) [inline src/UICircularProgressRing/UICircularProgressRing.swift]: I like the idea of this feature, however don't think it should be a property.
We should do something like

`startProgress(to: 100, inReverse: Bool)`

File: src/UICircularProgressRing/UICircularRingLayer.swift [hunk 2]
```diff
@@ -39,6 +39,7 @@ class UICircularRingLayer: CAShapeLayer {
     @NSManaged var value: CGFloat
     @NSManaged var minValue: CGFloat
     @NSManaged var maxValue: CGFloat
+	@NSManaged var isReverse: Bool
```
Comment (luispadron) [inline src/UICircularProgressRing/UICircularRingLayer.swift]: fix indentation please

File: src/UICircularProgressRing/UICircularRingLayer.swift [hunk 3]
```diff
@@ -400,10 +401,13 @@ class UICircularRingLayer: CAShapeLayer {
 
         // Draws the text field
         // Some basic label properties are set
+
+		let valueToDisplay = isReverse ? (maxValue - value) + 1 : value
```
Comment (luispadron) [inline src/UICircularProgressRing/UICircularRingLayer.swift]: indentation is wrong here

**Comment by luispadron** [comment]
Could you also add some tests to make sure `inReverse` works as expected.

**Comment by JoaquinMG** [comment]
Is there any way to countdown (from 10 to 0) instead of counting up (from 0 to 10)?

**Pull request closed without merging**